core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -86,6 +90,7 @@ public void execute(TaskListener listener) {
         listener.getLogger().println("Cleaned up "+numFiles+" records");
     }
 
+
```

Comment:
nit: I would try to avoid adding newlines unless it is directly adjacent to new code.